### ISSUE TITLE
Restore pre-trained model weights

### ISSUE THREAD

**Issue by steward1212** [endernewton/tf-faster-rcnn#231]
I would like to restore pre-trained model weights as my initial model weights. 

Thus, I followed your instruction and got the following message.

Loading initial model weights from data/imagenet_weights/res101.ckpt
Variables restored: resnet_v1_101/conv1/BatchNorm/beta:0
Variables restored: resnet_v1_101/conv1/BatchNorm/gamma:0
Variables restored: resnet_v1_101/conv1/BatchNorm/moving_mean:0
Variables restored: resnet_v1_101/conv1/BatchNorm/moving_variance:0
Variables restored: resnet_v1_101/block1/unit_1/bottleneck_v1/shortcut/weights:0
Variables restored: resnet_v1_101/block1/unit_1/bottleneck_v1/shortcut/BatchNorm/beta:0
Variables restored: resnet_v1_101/block1/unit_1/bottleneck_v1/shortcut/BatchNorm/gamma:0
Variables restored: resnet_v1_101/block1/unit_1/bottleneck_v1/shortcut/BatchNorm/moving_mean:0
Variables restored: resnet_v1_101/block1/unit_1/bottleneck_v1/shortcut/BatchNorm/moving_variance:0

All the restored variables value are 0.

Is it normal? Or I made some mistakes?

**Comment by endernewton** [comment]
where did you see it is 0? tensorboard?

**Comment by endernewton** [comment]
:0 is just a number to index the parameters..

**Comment by steward1212** [comment]
Got it.

Thank you.

**Comment by sulth** [comment]
@steward1212 ,Hi,How did you solved it.Iam also having the same responce:

**Comment by sulth** [comment]
Variables restored: resnet_v1_101/block4/unit_2/bottleneck_v1/conv3/BatchNorm/moving_variance:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/weights:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/BatchNorm/gamma:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/BatchNorm/beta:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/BatchNorm/moving_mean:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv1/BatchNorm/moving_variance:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/weights:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/BatchNorm/gamma:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/BatchNorm/beta:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/BatchNorm/moving_mean:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv2/BatchNorm/moving_variance:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/weights:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/BatchNorm/gamma:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/BatchNorm/beta:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_mean:0
Variables restored: resnet_v1_101/block4/unit_3/bottleneck_v1/conv3/BatchNorm/moving_variance:0
Loaded.
Fix Resnet V1 layers..
Fixed.
